Sources/MapboxNavigation/NavigationViewportDataSource.swift

```diff
@@ -476,8 +493,8 @@ public class NavigationViewportDataSource: ViewportDataSource {
                         return intersection.distance(to: intersectionLocations[index])
                     })
                     let filteredIntersectionDistances = intersectionDensity.enabled
-                        ? intersectionDistances.filter { $0 > intersectionDensity.minimumDistanceBetweenIntersections }
-                        : intersectionDistances
+                    ? intersectionDistances.filter { $0 > intersectionDensity.minimumDistanceBetweenIntersections }
```

Comment:
nit: I find the former variant easier to read.

Reply:
Agree, but Xcode considers this as a correct indentation.